src/transformers/training_args.py

```diff
@@ -1491,6 +1496,13 @@ class TrainingArguments:
         },
     )
 
+    use_liger: Optional[bool] = field(
+        default=False,
+        metadata={
+            "help": "Whether or not to enable the Liger (Linkedin GPU Efficient Runtime) Kernel for model training."
```

Comment:
```suggestion
            "help": "Whether or not to enable the Liger Kernel for model training."
```
The full name looks too verbose